### PR TITLE
Add Australian ICSM transformation grids

### PR DESCRIPTION
**Pull request by nyalldawson**
(Alternative version of https://github.com/OSGeo/proj.4/pull/708)

These files are available under a CC-BY-4.0 license from https://github.com/icsm-au/transformation_grids

### CODE REVIEW

**Comment by andrewharvey** [comment]
https://github.com/OSGeo/proj-datumgrid/pull/3/files#diff-423261653a17fd954e957602ebb5ce13L1 should be updated to include Australia or just removed to avoid a long list of counties in the title?

**Comment by nyalldawson** [comment]
I'd suggest removing it - it'll only get longer in future...

File: A66_National_13_09_01.LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Licensed under a Creative Commons Attribution 4.0 Licence.
```
Comment (kbevers) [inline A66_National_13_09_01.LICENSE.md]: I don't think these license files are needed. A reference to the publishing authoritys website should be enough. As far as I can tell it is this one https://github.com/icsm-au/transformation_grids and a clear statement in `GRIDS.md` which license is attached to each grid.

File: README.DATUMGRID [hunk 2]
```diff
@@ -12,3 +12,29 @@ CTable2 files are not system dependent (as distinct from the CTable files
 produced in PROJ 4.7.0 and older). 
 
 
+Australia	(sourced from https://github.com/icsm-au/transformation_grids)
```
Comment (kbevers) [inline README.DATUMGRID]: Please move this info to the `GRIDS.md` file instead.

**Review by kbevers — CHANGES_REQUESTED** [review]
I have a few change request in in-line comments. Fix those and we can include the Australian grids in version 1.7 of the datumgrid package which will have to be released together with PROJ 5.0.0.

**Comment by nyalldawson** [comment]
@kbevers PR updated

**Review by kbevers — APPROVED** [review]

**Pull request merged**